doc/sphinx/proofs/writing-proofs/reasoning-inductives.rst

```diff
@@ -525,31 +525,41 @@ This section describes some special purpose tactics to work with
 
 .. tacn:: discriminate {? @induction_arg }
 
-   Proves any goal for which a hypothesis in the form :n:`@term__1 = @term__2`
-   states an impossible structural equality for an inductive type.
-   If :n:`@induction_arg` is not given, it checks all the hypotheses
-   for impossible equalities.
-   For example, :g:`(S (S O)) = (S O)` is impossible.
-   If provided, :n:`@induction_arg` is a proof of an equality, typically
-   specified as the name of a hypothesis.
```

Comment:
This documentation change removed this sentence, making the documentation incomplete (there is no clear documentation of what this tactic does if `induction_arg` is given now). The sentence should be reintroduced.

Reply:
See #19623